verify-node/Makefile

```diff
@@ -0,0 +1,35 @@
+# 
+# Test and build Holo-config verify-node example
+# 
+#     Tests and wasm-packs a Node-compatible subset of holo-config's Config and Admin Key APIs,
+# sufficient to verify AdminPublicKey signatures on messages, from a NodeJS application.
+# 
+SHELL	= bash
+
+# nix-test, nix-build, ...
+nix-%:
+	nix-shell ../shell.nix --pure --run "make $*"
```

Comment:
This is not how `nix-shell` is supposed to be used.

Reply:
The Makefile is mostly a documentary repository; it contains examples of the imperative procedure for performing various actions.  This is the action required to achieve this objective; it must either be documented here (in executable, testable form), or documented in a README somewhere (in out-of-date, non-working form, usually)...

There's a reason that pretty much every historical Unix package *all* had: `./configure; make; make install` build procedure -- not because it was perfect, but because it was *documentation*!  I've seen a hundred attempts to throw this documentary convention aside, and "do it better".  They have *all* failed.

The first thing a new visitor must do, today, when arriving at any of our repos, is to ask: "How do I do X"?  Usually, the answer is nowhere to be found; they must just guess, or trawl the code to deduce how it "must" be done.  Usually incorrectly.

This makes it clear...